### ISSUE TITLE
Allow on-demand building the admin from `kinto-admin build` 

### ISSUE THREAD

**Issue by n1k0** [Kinto/kinto-admin#192]
Refs #189, #190 especially https://github.com/Kinto/kinto-admin/pull/190#discussion_r75626491

Right now the `kinto-admin build` command simply copies already built assets, which prevents to dynamically set specific options like the static assets url path.

The same problem exists with the current plugin system, where you cannot use this command to build an admin only enabling a given set of plugins (no mention of custom user-contributed ones).

We probably want to expose options to the command so we can contextually build the admin we actually want. That means not copying already built files anymore, but rather live-building them when the command is called. And having webpack and friends installed as package runtime dependencies.

**Comment by n1k0** [comment]
We should laternatively consider using [create-react-app](https://github.com/facebookincubator/create-react-app) and provide documentation how to install the kinto-admin on top of it.

**Comment by n1k0** [comment]
The more I think about it, the more I'm convinced we shouldn't try to build a customized admin ourselves, we should delegate the responsibility of doing this to the developer, especially now that we have tools like *create-react-app* which is likely to become a defacto standard to have any react app up and running in no time.

**Comment by n1k0** [comment]
Dream setup instructions:

```
$ npm install -g create-react-app
$ create-react-app mykwac && cd mykwac
$ npm install kinto-admin --save-dev
```

Then importing and rendering a `KintoAdmin` component:

```
// App.js
render(<KintoAdmin />, document.getElementById("main"))
```

We could obviously leverage JSX for configuration and possibly plugins as well. Then when ready,

```
$ npm start # or
$ npm run build
```

**Comment by n1k0** [comment]
> Dream setup instructions:

See #228 which implements just this.

**Comment by n1k0** [comment]
Closed by #228 and #229.
